roles/neutron-common/templates/etc/neutron/neutron.conf

```diff
@@ -109,6 +119,3 @@ rabbit_host = {{ endpoints.rabbit }}
 {% endif -%}
 rabbit_userid = {{ rabbitmq.user }}
 rabbit_password = {{ secrets.rabbit_password }}
-
-[oslo_middleware]
-enable_proxy_headers_parsing = True
```

Comment:
Why was this removed? This is needed for proxy filtering to make sure http becomes https.